Documentation/Signoffs/jdjoyce42-power-signoff.md

```diff
@@ -0,0 +1,188 @@
+# Power Subsystem
+ 
+## Function of the Subsystem
+
+The purpose of this subsystem is to take the voltage supplied from a wall outlet and make it usable for the project. This will be done by supplying each subsystem with its respective DC voltages.
+
+
+## Constraints
+
+| No. | Constraints                                                                         | Origin            |
+| --- | ----------------------------------------------------------------------------------- | ----------------- |
+| 1   | System shall convert wall AC Voltage to DC voltage | Design Constraint |
+| 2   | Shall be capable of supplying a minimum of 29 W                       | Design Constraint |
+| 3   | Shall be able to step down power supply voltage to 5 VDC                            | Design Constraints |
+| 4   | System shall be powered by a standard wall outlet                                   | Design Constraint |
+| 5   | System shall be grounded and not have live wire exposed                                    | OSHA and IEC Safety Standards  |
+
+
+
+<sup>1</sup> The system will need to convert VAC to VDC for each device to be functional. A small margin of error is expected due to unideal power supply ripple and other nonlinear effects.
+
+<sup>2</sup> A good rule of thumb when it comes to power supply is to supply at least 1.2x the max power consumption of the system [9]. This helps account for any fluctuations that might happen in the power draw. To make this number even safer, a constraint of a 29 Watt minimum has been placed on the device. This is equal to 1.5x the max power draw total calculated from all of the systems. Ensuring enough power is constantly available.
```

Comment:
This is a good rule. How were the values determined for the power draws of each of the associated systems?